src/SFA.DAS.Commitments.Api/Orchestrators/EmployerOrchestrator.cs

```diff
@@ -407,5 +405,21 @@ private List<Domain.Entities.ProviderPaymentPriorityUpdateItem> CreateListOfProv
                 PriorityOrder = x.PriorityOrder
             }).ToList();
         }
+
+        public async Task<GetEmployerAccountSummaryResponse> GetAccountSummary(long accountId)
+        {
+            _logger.Trace($"Getting account summary for employer account {accountId}", accountId: accountId);
+
+            var response = await _mediator.SendAsync(new GetEmployerAccountSummaryRequest
+            {
+                Caller = new Caller
+                {
+                    CallerType = CallerType.Employer,
+                    Id = accountId
+                }
+            });
+
```

Comment:
We've been putting an Info log at the end of the Orchestrator calls. Give us opportunity to 1. log that the operation completed and 2. log info about the result of the call. e.g. number of apprenticeships returned or some other item of interest.